freddie/viewsets/sql.py

```diff
@@ -58,18 +58,25 @@ class GenericModelViewSet(GenericViewSet):
     model: Type[Model] = Model
     pk_field: DBField
     secondary_lookup_field: Optional[DBField] = None
+    model_ordering: Tuple[DBField] = ()
```

Comment:
why do you need class-level attribute, if you always override it in `__init__`?

Reply:
My bad. 
But I think it would be a good feature to have ability to define it in both places. So I suggest:
```
def __init__(...)
    ...
    if model_ordering:
        self.model_ordering = model_ordering
    ...
```

And if it is not necessary in `__init__` I can simply remove it.

What do you prefer?

Reply:
lgtm

Reply:
Done